src/vtquery.cpp

```diff
@@ -19,32 +19,26 @@ namespace VectorTileQuery {
 
 struct ResultObject {
     ResultObject(
```

Comment:
This constructor probably needs to use moves where possible, currently it can make several copies. If you want to chat about the differences on voice we can do that.